1 - identify organisations in the corpus/ner_flair.ipynb

```diff
@@ -1,25 +1,59 @@
 {
```

Comment:
<p>Here, it throws an error because the <code>text_processing</code> package is not installed. It should be added to <code>requirements.txt</code>. [Required]   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/disaster-capitalism/named-entities/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>On google colab, it also doesn't find the <code>text_processing</code> moduel even when it is installed (it might be beause colab uses Python 3.7 for which the module has a different name?). </p>

Reply:
It is added already... I'm not sure what the issue here is. Python does not allow dashes in package names. And the library is called "text-preprocessing". But when you import it, you use "text_preprocessing". I added "text_preprocessing" to the requirements.txt. Hopefully that solves it for you. It works fine on Google colab for me though...

Reply:
Using Python 3.9 fixed it!